src/mscorlib/shared/System/Text/StringBuilder.cs

```diff
@@ -1645,6 +1645,36 @@ public bool Equals(StringBuilder sb)
             }
         }
 
+        /// <summary>
+        /// Determines if the contents of this builder are equal to the contents of ReadOnlySpan<char>.
+        /// </summary>
+        /// <param name="value">The ReadOnlySpan<char>.</param>
+        public bool Equals(ReadOnlySpan<char> value)
+        {
+            if (value.Length != Length)
+                return false;
+
+            StringBuilder thisChunk = this;
+            int thisChunkIndex = thisChunk.m_ChunkLength;
+
+            for (int i = Length - 1; i >= 0; i--)
+            {
+                --thisChunkIndex;
```

Comment:
I think we are generally using postfix style, for consistentcy (`thisChunkIndex--`). The STringBuilder code is old.

Of course when not used in an expression, the result is the same. 
It seems when we do use in an expression we almost exclusively use postfix.